src/sql/src/func.rs

```diff
@@ -2336,6 +2346,21 @@ lazy_static! {
                 params!(Float64, Float64) => ModFloat64, oid::OP_MOD_F64_OID;
                 params!(Numeric, Numeric) => ModNumeric, 1762;
             },
+            "&" => Scalar {
+                params!(Int16, Int16) => BitAndInt16, 1884;
```

Comment:
I think for `smallint` it should be `1874`

Reply:
Good catch.